apps/python-sdk/firecrawl/v2/methods/vector_search.py

```
@@ -0,0 +1,159 @@
+"""
+Vector search functionality for Firecrawl v2 API.
+"""
+
+from typing import Dict, Any
+from ..types import VectorSearchRequest, VectorSearchData, VectorSearchTiming, VectorSearchResult
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)